docs/reference-guides/block-api/block-patterns.md

```diff
@@ -1,23 +1,33 @@
 # Patterns
 
-Block Patterns are predefined block layouts, ready to insert and tweak.
+Block Patterns are predefined block layouts, available from the patterns tab of the block inserter. Once inserted into content, the blocks are ready for additional or modified content and configuration.
 
-## Block Patterns Registration
+In this Document:
+* [`register_block_pattern`](#register_block_pattern)
+* [`unregister_block_pattern`](#unregister_block_pattern)
+* [`register_block_pattern_category`](#register_block_pattern_category)
+* [`unregister_block_pattern_category`](#unregister_block_pattern_category)
+
+## Block Patterns
 
 ### register_block_pattern
```

Comment:
Would it make more sense to use the function description as the title rather than the function itself?

Reply:
For instance **Register block patterns**

Reply:
> This was not criticism but genuine surprise. I did not check all default themes but Twenty Nineteen and Twenty Twenty-One does not use a hook. I do not know what plugins need.

No worries, I didn't take it as criticism! It's true that the extra function isn't needed, I really thought it was. I will update the doc to reflect this. Interestingly, by omitting hooking into init the registered category moves to the top of the list instead of the bottom, I assume because it's actually firing the code before the core patterns load.